review-tools/ghmerge

```diff
@@ -60,7 +60,11 @@ while [ $# -ne 0 ]; do
         WHAT=web ; BUILD=no ; shift
         ;;
     --cherry-pick)
-        shift; PICK=$1; shift
+        shift;
+        PICK=1;
+        if [ $1 != "" ] && [ $1 -ge 1 ] && [ $1 -le 99 ]; then
```

Comment:
This will probably emit errors if the argument after --cherry-pick is a non-numeric

Reply:
You can add 2>/dev/null to the test statements  with -ge and -le to eliminate this.

Reply:
Actually it would be sufficient to add it to the -ge test call as the other won't be run.

Reply:
Right; fixed.